goodreads-predictor/src/goodreads_predictor/pipelines/feature_engineering/nodes.py

```diff
@@ -231,3 +238,50 @@ def apply_book_attributes(input_df: pd.DataFrame) -> Tuple[pd.DataFrame, Dict[st
     output_df['language_code'] = consolidated_language_codes
 
     return output_df, cutoff_dict
+
+# Defin ethe function to merge the output_df with the description embedding from catalog
+def merge_description_embeddings(output_df: pd.DataFrame, description_embeddings: pd.DataFrame) -> pd.DataFrame:
+    """
+    Merge the description embeddings with the output DataFrame.
+    Args:
+        output_df (pd.DataFrame): The output DataFrame.
+        description_embeddings (pd.DataFrame): The description embeddings DataFrame.
+
+    Returns:
+        pd.DataFrame: The merged DataFrame.
+    """
+    # Merge the description embeddings with the output DataFrame
+    merged_df = pd.merge(output_df, description_embeddings, on='isbn13')
```

Comment:
Hey, there's a field called DescriptionISBN in the books data after preprocessing. We will want to merge on that since we are not still at the ISBN13 level